force-app/main/default/classes/VariablesDataTypesOperators.cls

```diff
@@ -242,8 +304,8 @@ public with sharing class VariablesDataTypesOperators {
         final Double SUBTRACT_FACTOR = 32.0;
         final Double MULTIPLY_FACTOR = 5.0;
         final Double DIVIDE_FACTOR = 9.0;
-    
-        return null; // Replace null with the variable you used to store the result
+        Double celccalc = (fahrenheit -32) * 5/9;
```

Comment:
You can use the constants provided above on lines 304-306 instead of hardcoding the integers in the formula